src/background.js

```diff
@@ -20,7 +28,7 @@ function runShowMeTheCode(tabId, tab) {
     };
 
     if (isGitHub()) {
```

Comment:
Is it worth having separate if for isGitHub ? you can as well squash it all into single if, with isGitHub as its first check.